lasagne/layers/special.py

```diff
@@ -6,12 +6,13 @@
 from ..utils import as_tuple
 from .base import Layer, MergeLayer
 
-
 __all__ = [
     "NonlinearityLayer",
     "BiasLayer",
     "InverseLayer",
     "TransformerLayer",
+    "ParametricRectifierLayer",
+    "prelu"
```

Comment:
This still needs to end with a comma (as I already mentioned a week ago: https://github.com/Lasagne/Lasagne/pull/446#discussion_r41035676).